react-app/src/components/common/DateTime/UTCDatetime.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { format } from "date-fns";
+
+const UTCDatetime: React.FC<{ date: Date; className?: string }> = ({
+  date,
+  className = "",
+}) => {
+  return (
+    <time className={className} dateTime={date.toISOString()}>
+      {format(date.toISOString().slice(0, -1), "YYYY-mm-DD HH:mm")} UTC
```

Comment:
I think `format` takes `Date` as the arg you can just do 
```ts
format(date, "your desire format")
```

Reply:
This (`toISOString`) was originally intended to convert local time to UTC time, coz I thought the api will return a local js date object.

I should remove `toISOString` if the api returns a UTC datetime string directly, but I am not sure what the API date type is like.